### PR TITLE
[Booking_Service] models added

### PR DESCRIPTION
**Pull request by fedorovychh**
- accommodation
- address
- booking
- payment

### CODE REVIEW

**Review by DevIgork — APPROVED** [review]
💎

**Review by BohdanTrue — APPROVED** [review]

File: src/main/java/com/application/bookingservice/model/Accommodation.java [hunk 1]
```diff
@@ -1,4 +1,49 @@
...
+    @Column(nullable = false)
+    private Type type;
+    @OneToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "address_id", nullable = false)
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    private Address address;
+    private String size;
+    private String amenities;
+    private BigDecimal price;
```
Comment (nshtykh) [inline src/main/java/com/application/bookingservice/model/Accommodation.java:40]: nullable = false

File: src/main/java/com/application/bookingservice/model/Booking.java [hunk 2]
```diff
@@ -1,4 +1,53 @@
...
+@Entity
+@Data
+@Table(name = "bookings")
+@SQLDelete(sql = "UPDATE accommodations SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted=false")
 public class Booking {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private LocalDate checkIn;
```
Comment (nshtykh) [inline src/main/java/com/application/bookingservice/model/Booking.java:30]: Maybe @CreationTimestamp is needed here?

**Review by nshtykh — APPROVED** [review]
GJ!

**Pull request merged**